components/brave_wallet/browser/solana_transaction_unittest.cc

```diff
@@ -597,53 +597,6 @@ TEST_F(SolanaTransactionUnitTest, GetSerializedMessage) {
             std::make_pair(*expected_message_bytes2,
                            std::vector<std::string>(
                                {kFromAccount, kTestAccount, kToAccount})));
-
-TEST_F(SolanaTransactionUnitTest, GetSignedTransactionBytes) {
```

Comment:
Actually can you update this test to match current GetSignedTransactionBytes?
Should also add a check for empty signature in that function too which I missed.